source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet

```diff
@@ -48,6 +48,26 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
       ],
     },
     instruction: 'Tell the respondent to turn to <strong>Showcard 1</strong> or show them the Electronic Showcard below',
+    definitions: [
+      {
+        title: 'Electronic Showcard',
+        contents: [
+          {
+            description: '<strong>“Usually living at” means:</strong>',
```

Comment:
This should use `title` rather than `description` and the `<strong>` tags dropped - `title` elements are wrapped in `<strong>` tags in runner.

Reply:
Sorted, somehow I missed that, thanks👍